packages/ui-app/src/util/shouldDisplaySection.ts

```diff
@@ -0,0 +1,31 @@
+// Copyright 2017-2018 @polkadot/ui-app authors & contributors
+// This software may be modified and distributed under the terms
+// of the ISC license. See the LICENSE file for details.
+
+import isUndefined from '@polkadot/util/is/undefined';
+import isNull from '@polkadot/util/is/null';
+
+function anyMethodToDisplay (methods: any) {
```

Comment:
Almost there. `any` is a bit of a "smell", only used when no options. Here we can actually type it using Section + generics. (The callers all have the same generic-ified Section type)